### ISSUE TITLE
Audio delay using sndcpy

### ISSUE THREAD

**Issue by kumarraviraushan71** [rom1v/sndcpy#275]
operating system Mac OS m1 chip 
after successfully staring sndcpy in terminal , I first get 
main input error: ES_OUT_SET_(GROUP_)PCR  is called too late (pts_delay increased to 0ms)
after some time means 1 or 2 minutes it just keep increasing causing a delay in audio:
something like this and this is just unacceptable
main input error: ES_OUT_SET_(GROUP_)PCR  is called too late (pts_delay increased to 104ms)
main input error: ES_OUT_SET_(GROUP_)PCR  is called too late (pts_delay increased to 377 ms)

is there any fix to solve this issue where I can somehow maintain may audio delay to 0 ms  I have preinstalled VLC 
if possible then plzz solve

**Comment by Kinuseka** [comment]
Have you checked the solution proposed on issue #154 ? 

It works on my case

**Comment by rom1v** [comment]
There is no latency regulation in sndcpy, it was just a proof-of-concept. Audio is properly integrated in scrcpy now (for Android 11+).

**Comment by Kinuseka** [comment]
Yeah, however its not really a solution in my case as I am using a device on Android 10 which is unfortunate, but fortunate enough that a repo like this exist. The Proof of concept works great in my own case, and its still a great alternative over native scrcpy (for android 10) 

On this particular issue the latency was caused by VLC network caching which causes delay. adding `--network-caching=-1` disables the caching and allows <100ms delay
